src/univariate/continuous/generalizedextremevalue.jl

```diff
@@ -5,7 +5,7 @@ The *Generalized extreme value distribution* with shape parameter `ξ`, scale `
 
 ```math
 f(x; \\xi, \\sigma, \\mu) = \\begin{cases}
-        \\frac{1}{\\sigma} \\left[ 1+\\left(\\frac{x-\\mu}{\\sigma}\\right)\\xi\\right]^{-1/\\xi-1} \\exp\\left\\{-\\left[ 1+ \\left(\\frac{x-\\mu}{\\sigma}\\right)\\xi\\right]^{-1/\\xi} \\right\\} & \\text{for } \\xi \\neq 0  \\\\\\
```

Comment:
With this PR one can actually read something
![image](https://user-images.githubusercontent.com/765740/118317304-c7f77980-b4ef-11eb-9054-266e11438a82.png)
instead of
![image](https://user-images.githubusercontent.com/765740/118317396-ebbabf80-b4ef-11eb-9786-ae3520260955.png)